pkg/pillar/cmd/downloader/downloader.go

```diff
@@ -1190,6 +1357,13 @@ func handleSyncOp(ctx *downloaderContext, key string,
 		log.Fatalf("handleSyncOp: No ObjType for %s\n",
 			status.Safename)
 	}
+
+	log.Debugf("handleSyncOp: config: %+v", config)
```

Comment:
\n

Reply:
I think it implicitly adds a \n ( log.Debugf )